modules/social_features/social_profile/modules/social_profile_organization_tag/social_profile_organization_tag.install

```diff
@@ -157,3 +159,44 @@ function social_profile_organization_tag_update_130001(): void {
   user_role_grant_permissions('contentmanager', ['create terms in profile_organization_tag']);
   user_role_grant_permissions('sitemanager', ['create terms in profile_organization_tag']);
 }
+
+/**
+ * Create entity-form-display or hide path field from Organization Tag taxonomy.
+ */
+function social_profile_organization_tag_update_130002(): void {
```

Comment:
Can you explain why this update needs to be done using update hook and not form_alter?

Reply:
The Acceptance Criteria is asking to change using configuration, because that way can reverted easily by other system if necessary.